Chinese/fighting.md

```diff
@@ -0,0 +1 @@
+金城盃打架事件後，台灣媒體立刻針對北大球員打人報導，報導內容多對北大球員的攻擊台藝大選手做描述，引發北大球員不滿，北大球員郭凱在微
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.